drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -3099,7 +3099,7 @@ void RasterizerStorageGLES3::_update_material(Material *material) {
 		}
 
 		glBindBuffer(GL_UNIFORM_BUFFER, material->ubo_id);
-		glBufferSubData(GL_UNIFORM_BUFFER, 0, material->ubo_size, local_ubo);
+		glBufferData(GL_UNIFORM_BUFFER, material->ubo_size, local_ubo, GL_STATIC_DRAW);
```

Comment:
I noticed that this is the only call using `GL_STATIC_DRAW` instead of `GL_DYNAMIC_DRAW`, is this on purpose or an oversight?

Reply:
On purpose. The buffer was allocated a few lines above with GL_STATIC_DRAW. We match the access type at the time of creation. This is only a hint anyway.